publiccode.yml

```
@@ -150,7 +130,7 @@ maintenance:
 localisation:
   localisationReady: no
   availableLanguages:
-    - ita
+    - it
 
 dependsOn:
   open:
```

I would change this to `proprietary` as IIS is not open source:

```suggestion
  proprietary:
```

---

@alranel check line 161

---

Hi @libremente. Thanks a lot for your highly appreciated support.

This software is written under the AGPLv3 license. It is not only open-source, it is even Free Software, according to the GNU definition.
It is true that is can be run under IIS which is proprietary software. But it can be easily run under [MONO framework](https://www.mono-project.com/), as well, which is a completely open source porting of the .NET framework, running under Linux, MacOS and Windows operating systems.

So, the software can be operated by using only open-source tools.

Looking forward to hear for your feedback, before merging the pull-request. Thanks.

---

Whoops, my bad. I didn't expand the collapsed lines in GitHub diff :-) I thought IIS was listed under `open`, thus my comment.